examples/rafmekerk_signaling/logLikelihoodRMELsqnonlin.m

```diff
@@ -0,0 +1,81 @@
+ function [varargout] = logLikelihoodRMELsqnonlin(theta, amiData)
+% compute the negative loglikelihood and gradient for RafMekErk data
+
```

Comment:
Please document input parameters and especially `varargout`. State difference to /logLikelihoodRME.m